examples/proto_bench/generate.bash

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Currently generates two test files
+# + _autogen_many_sym.lp
```

Comment:
Why starting with _?

Reply:
Just to mark the autogen files, there is no particular reason.  It is rather silly given that they are already tagged 'autogen'